module/src/utils.ts

```diff
@@ -0,0 +1,72 @@
+// ** utility functions **
+
+import { LatLngTuple, GeoBounds, Position, QueryParameters } from './types'
+
+// ** check position is in bounds
+export function inBounds(position: LatLngTuple, bounds: GeoBounds): boolean {
+  if (position && typeof position[0] == 'number' && typeof position[1] == 'number') {
+    return position[1] >= bounds.sw[1] &&
```

Comment:
This will not work if the bounds cross the dateline. I wonder if there is a ready made solution that would work there? I am aware of turf.js, but not sure if it works over the dateline.

Reply:
Because the bbox is lower left -> upper right this makes dateline crossing handling a little more straight forward.
Added logic to detect a lon + -> - transition and also modify position lon accordingly if the bbox crosses the dateline.